WeatherNotFound/src/main/java/me/learning/weathernotfound/presentation/WeatherNotFound.kt

```diff
@@ -78,7 +79,14 @@ class WeatherNotFound private constructor() {
 
         setInitCalledState(true)
 
-        OPEN_WEATHER_API_KEY = openWeatherApiKey
+        OPEN_WEATHER_API_KEY = context.packageManager
+            .getApplicationInfo(context.packageName, PackageManager.GET_META_DATA)
+            .metaData
+            .getString(OPEN_WEATHER_TOKEN_MANIFEST_KEY)
+            ?: throw IllegalArgumentException(
+                "$OPEN_WEATHER_TOKEN_MANIFEST_KEY doesn't found in manifest. " +
+                        "Please provide it as metadata in your application manifest"
+            )
         // Todo: For now, SDK can just support Json responses! Other formats will be added soon.
```

Comment:
Please create a function for this operation and return just a `String` as the result. I believe it will help the developer understand code more easily.